src/SearchParameters/index.js

```diff
@@ -325,6 +325,12 @@ function SearchParameters(newParameters) {
    * @member {string}
    */
   this.insideBoundingBox = params.insideBoundingBox;
+
+  forEach(params, function checkForUnknownParameter(paramValue, paramName) {
+    if (!this.hasOwnProperty(paramName)) {
+      throw new Error('algoliasearch-helper: unknown parameter: `' + paramName + '`');
```

Comment:
I would not prefix the error message with `algoliasearch-helper` (and we don't do it for other errors).